### PR TITLE
fix(tests): storm client test case

### PR DESCRIPTION
**Pull request by numb3r3**
Start flow
```bash
$ python python tests/storm/echo_flow.py [port] [--parallel] [--protocal] [--polling]
```

Then, run the storm test:
```bash
$ python tests/storm/storm_client.py [port] [--protocal] [--concurrent]
```

**Note**: all of the arguments have default value, so you can start the testing without providing any options.

**NEW UPDATES**
```
$ pytest  tests/integration/concurrent_clients/
```

### CODE REVIEW

File: tests/storm/storm_client.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import click
```
Comment (bwanglzu) [inline tests/storm/storm_client.py:1]: `click` does not exist in our `requirements` nor `extra-requirements.txt`

**Review by bwanglzu — COMMENTED** [review]
what is `storm`? maybe you can move it to `integration`?

File: tests/storm/storm_client.py (hunk 1 above)
Comment (JoanFM) [reply tests/storm/storm_client.py:1]: can u turn them into `pytest` tests with proper parametrization? @numb3r3

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: tests/storm/storm_client.py (hunk 1 above)
Comment (numb3r3) [reply tests/storm/storm_client.py:1]: @JoanFM sure, i will convert this as pytest cases

**Review by soljarjb13 — APPROVED** [review]

**Review by soljarjb13 — APPROVED** [review]

**Review by soljarjb13 — APPROVED** [review]

**Comment by soljarjb13** [comment]
v

**Comment by soljarjb13** [comment]
v

**Review by soljarjb13 — APPROVED** [review]

**Comment by numb3r3** [comment]
this PR has been merged into https://github.com/jina-ai/jina/pull/2843

**Pull request closed without merging**